### ISSUE TITLE
Can't bring out welder while running server with SEServerExtender

### ISSUE THREAD

**Issue by KodakYarr** [Tyrsis/SE-Community-Mod-API#58]
When I run a dedicated server with SESE neither me nor anyone else can use the welder tool. It's not selectable from the tool bar.
I have tried reinstalling .NET 4.5. I have tried different worlds. I have tried the tool in any slot. I have also tried in another instance.
The other tools including the rifle works; it's just the welder. When I press the button for the welder the toolbar slot blinks and I hear the clink sound as if it was selected, but then immediately deselected.

https://github.com/SEModCommunity/SE-Community-Mod-API/issues/163
This similar ticket for the older project mentions all the tools not working and that Czech might be a likely culprit. I'm running Windows 7 Ultimate English; nothing Czech here.

I can't seem to find any error logs about it when running the server.

**Comment by Traxzorg** [comment]
I've had the same problem also and with a English install of windows 8.1. I'm not sure if this is what solved it, but I went to the control panel -> region and changed format, location and Language for non-unicoide programs from Swedish to English. Restarted the computer and then the welder worked. 

Hope that helps.

**Comment by KodakYarr** [comment]
Thanks for the suggestion. I thought I had tested changing this and was pretty sure I had already, but I guess I must have missed something... Trying your suggestion again I found the reason why the welder wouldn't come out while using SE Server Extender. Turns out it doesn't like **Swedish (Sweden)** as the *Format* in the *Region and Language* settings in Windows, and it's now set to **English (United Kingdom)** but with all the settings manually changed to how we have it in Sweden. :)

I used the exclusion method and found that it was the **Format** settings alone, and nothing to do with any of the other tabs in *Region and Language*, that caused the problem. I suppose it's something to do with the Swedish words for weekdays, some of which that have special characters like å and ö.
